manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: CologneCodeCompany.XYplorer
+PackageVersion: 26.20.0200

```

```suggestion
PackageVersion: 26.20.0203
```
Version should match what is written to control panel. After changing this, you will need to update the name of the version folder

---

What's meant here is really the Registry. Both the Control Panel's Add and remove programs page (aka ARP) and the Apps and features Settings page (aka ANF) display the Registry data for installed applications. And this data can be retrieved with other means, including the Registry Editor. 

Does the application display a different version number? 

Sometimes, an application will be written to display one version number, or the version number in one format - while the application's installer is configured to write a different version number, or the version number in another format, to the Registry. When this happens, the usual solution is adding a `DisplayVersion` to match the Registry data. This way, the package manager can display a version number that matches the application, by matching repo data with Registry data. 

I haven't installed the application, so I'm not sure if this is occurring here. But it sounds like the application might display `26.20.0200`, while the installer writes `26.20.0203` to the Registry. Is this the case?